olm/eclipse-che-preview-kubernetes/Dockerfile

```diff
@@ -5,10 +5,10 @@ RUN ./bin/initializer -o ./bundles.db
 
 FROM openshift/origin-base
 
-COPY --from=builder /build/bin/initializer /initializer
-COPY --from=builder /build/bin/configmap-server /bin/configmap-server
-COPY --from=builder /build/bundles.db /bundles.db
-COPY --from=builder /build/bin/registry-server /registry-server
+COPY --from=builder /bin/initializer /initializer
```

Comment:
To be honest I don't understand why `/build` isn't necessary anymore. Could you explain pls?

Reply:
@AndrienkoAleksandr

Reply:
During my testing, running `quay.io/operator-framework/upstream-registry-builder` had moved it's artifacts out of `/build`

Reply:
We could tag to a specific image

Reply:
Yes, this image will be updated and pinned https://github.com/eclipse/che-operator/pull/522 in the frame of the issue: che#18105

Reply:
that is the `upstream-opm-builder` image, not the `upstream-registry-builder`.  If `upstream-registry-builder` is deprecated in favor of `opm`, this will need to be updated anyway.  How often are these catalogs built with this Dockerfile?  Probably not too often, as I think you would have seen these fail by now.

Reply:
Oh, sorry, I thought about another Dockerfile. This one we really could pin.